### DISCUSSION TITLE
Editting Repository name

### DISCUSSION THREAD

**Discussion by JumaMinya** [IBAHCM/RPiR#463] [Archive]
@IBAHCM/rpir-instructors 

I have mistaken created this repository
https://github.com/IBAHCM/JumaMinyaBCT.git

Can I have access to correct the name "JumaMinyaBCT" to "JumaMinyaBCI" since I have transferred ownership that goes with setting access.

**Comment by soniamitchell** [comment]
Done 👍

**Comment by JumaMinya** [comment]
@soniamitchell 
Thanks
